### DISCUSSION TITLE
Помогите пожалуйста найти прошивку или дамп для этой камеры, нигде ничего нет не могу найти

### DISCUSSION THREAD

**Discussion by 0003426085** [e27-camera-hack/E27-Camera-Hack#2] [Q&A] [answered]
Здравствуйте **
![IMG_20230815_204056](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/36a194d5-bd86-4a4e-8320-a096917ae306)
![IMG_20230815_204049](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/dc7eca53-6e29-4dc0-b64a-dfa94582fbbd)
![IMG_20230815_204032](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/1355cd3b-4d38-4b7f-aad3-e62f64369469)
![IMG_20230815_203845](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/85010a39-4ef3-4e4c-9012-118cd11a20bb)
![IMG_20230815_203958](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/50b5f59c-88ce-4ec4-9580-15d318312a2e)
![Picture_20230815081136](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/102b6d2b-6c55-4ab8-8128-b2388ccf2e88)
![IMG_20230713_192753](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/8c8bbc98-2cef-45ac-858a-7ffc2e89d456)
![Picture_20233413073440](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/ce3c7e49-6e48-443f-8a65-fe8a24fead7d)
![Picture_20233313073302](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/c0b8f9ec-e595-43b9-ba46-e5812a3a3b98)
![IMG_20230713_183446](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/f02e2cf2-f57d-4c55-b937-6a9ca5f96c6a)
![IMG_20230713_193517](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/c76c3f41-01d0-41df-98a4-412a801ad3c0)
Помогите пожалуйста найти прошивку или дамп. для этой камеры, нигде ничего нет не могу найти**

**Comment by filder35** [comment]
What exactly doesn’t work for you and why you need the firmware. First, I advise you to merge your original dump (if something goes wrong). If you have any questions, ask.

**Reply by 0003426085** [reply]
Здравствуйте, Наконец то хоть кто-то ответил за год, заранее благодарю,дамп я слил сразу. Камера не загружается, светится только подсветка. Но я заливал какую-то прошивку и камера, что-то начала говорить и всё и тоже дальше не грузится.

**Reply by filder35** [reply]
Случайно зашел в дискусии. До этого мы все с версией 200 разбирались . У вас версия 300 судя по процессору. На фото вижу лог загрузки. Как вы его снимали . И дамп через что слили. У меня нет камеры версии 300 но алгоритмика работы у них одинаковая. "какую-то прошивку "  какую можно посмотреть если сохранилась. Если светиться значит загрузчик жив.  Надо смотреть на чем она останавливается.

**Reply by 0003426085** [reply, marked as answer]
Дамп слил китайским программатором CH341A, лог UART модуль PL2303HX . Дамп залил такой 
[AK3918EV300_full_dump.zip](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14477664/AK3918EV300_full_dump.zip)
Ой Я извиняюсь писал, что нашёл прошивку,а это дамп залил его программатором и камера ведёт себя так, как на видео 

https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/b55cb5fc-9c73-4a84-9ca6-9bc6b78f6d67

**Reply by 0003426085** [reply]
И перезагружается

**Reply by filder35** [reply]
наверное не выпаивали флеш (прищепкой)? Можете снять лог загрузки в том состоянии в каком она сейчас. Логи много информации дают.

**Reply by 0003426085** [reply]
Нет Я выращивал. Вот журнал загрузки 
[лог 3агру3ки.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14479927/3.3.txt)
И видео 

https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/f9587fd9-917e-4602-8c17-eba1db722da4

После камера перезагружается и так по кругу

**Reply by filder35** [reply]
Пока только вижу что камера сенсор потеряла остальное все вроде живое.  Другая прошивка вам не поможет . Если только с такого же идентичного устройства. Надо пробовать эту восстановить . Попробую дамп разобрать посмотреть что там и как лежит.

**Reply by 0003426085** [reply]
Спасибо ВАМ Большое. Может Вам лучше дамп который Я с этой Камеры слил
[МОЙ ДАМП.zip](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14480393/default.zip)

**Reply by 0003426085** [reply]
Ещё раз снял лог,может тут лучше видно. СПАСИБО ВАМ Большое

**Reply by 0003426085** [reply]
[ЛОГ 3АГРУ3.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14480446/3.3.txt)

**Reply by filder35** [reply]
Поверхностно глянул дампы они разные по структуре. Но камера грузиться это уже хорошо. К старому варианту работоспособности камеры вернуться уже возможно не сможем надо пробовать восстанавливать . Вы как с линуксом представление имеете. Многие операции надо будет производить под линуксом.
Это примерный вариант.
1. Надо будет остановить загрузку и загрузиться /bin/sh
2. произвести изменения в исполняемых файлах (остановить приложения которые приводят к перезагрузке, включить телнет, ФТП, поменять варианты загрузки под свои нужды.
3. Осмотреться на наличие файлов отвечающих за запуск оборудования вифи сенсора. 
4. Пробовать их запускать в ручном режиме
Вот на пока все. Дальше как пойдет. Все делается на свой страх и риск (крайний вариант выпаивать флеш но много раз ее паять не рекомендую платы нежные можно навсегда убиить.)

**Reply by 0003426085** [reply]
Спасибо ВАМ большое,но к сожалению Я с Линуксом не умею работать. Чтобы хотя бы видео посмотреть, как с ним работать

**Reply by 0003426085** [reply]
Здравствуйте, а больше ничего не можно сделать

**Comment by filder35** [comment]
почему можно попробовать.  Вы подключены к камере по  UART ?

**Reply by 0003426085** [reply]
Да Я подключился по UART

**Reply by 0003426085** [reply]
Извиняюсь, что долго не отвечал, только увидел

**Comment by filder35** [comment]
Исследуя ваши дампы они абсолютно разные. Один который AK3918EV300_full_dump.bin я так понял вы им обновились . Второй (я так понял это оригинал) либо битый либо блок в котором лежат файлы и драйвкра к оборудованию под ключом. Попробую еще его разобрать. 
А так для начала пробуйте при включение к питанию камеру нажимать Ctrl+C .
 Это чтобы остановить загрузку системы.
Должно появится что то вроде  anyka#  Это вы попадете в командную строку загрузчика.
Дальше наберите help  - меню команд которые поддерживает загрузчик
Потом наберите  printenv   - это переменные которые установлены для запуска 
Выложите их сюда будем смотреть. Это для начала.
И еще вопрос флешку вы выпаивали или нет? Как вы сняли родной дамп?

И не торопитесь делайте все осознанно внимательно .

**Reply by 0003426085** [reply]
Вот файл, а флешку Я выпаиевал феном и 
[файлы.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14511705/default.txt)
снял программатором дамп

**Reply by 0003426085** [reply]
Спасибо что помогаете

**Reply by 0003426085** [reply]
Да а этим AK3918EV300_full_dump.bin обновился

**Reply by filder35** [reply]
Да не за что сам когда то с убитой камеры начинал. 
Есть фен и опыт выпаивния флешь это уже хорошо.
Оригинал тоже пробовали заливать ?  
Сильно на плате тоже не желательно паяться (платы нежные ноги отрываются) только когда уже вариантов нет.
Я сначала составлю инструкцию действий .Но вернуться к тому состоянию в котором камера была может не получиться но пробовать надо.
И еще вопрос СД в камере есть?
В командной строке загрузчик проверьте  mmcinfo либо  mmc
Если есть СД флешка вставьте ее и включите камеру лог сюда выложите (проверить система видит СД) - это дополнительный вход в камеру для дальнейших манипуляций.

**Reply by 0003426085** [reply]
Да оригинальный дамп заливал, камера вообще не работает светится подсветка. Да камеру хоть как нибудь оживить, вот лог
[mmcinfo1.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14512236/mmcinfo1.txt)

**Comment by filder35** [comment]
В командной строке загрузчика

setenv init /bin/sh
run setcmd
saveenv
reset

Теперь вы загрузились в командную строку как root без входа в систему.

Пробуем меняем пароль

passwd root

если сработает введите свой пароль потом еще раз
(у меня это не сработало я менял файл)

Теперь мы вручную монтируем остальную часть файловой системы согласно /etc/init.d/rcS. которая у вас в камере
каждую команду по отдельности вводим

echo "mount all file system..."
mkdir /dev/pts
/bin/mount -av
umask 022
export runlevel prevlevel
echo "starting mdev..."
/bin/echo /sbin/mdev > /proc/sys/kernel/hotplug
mdev -s

mkdir /var/cache
mkdir /var/run
mkdir /var/log
mkdir /var/spool

далее монтируем файловую систему так же каждую команду по отдельности вводим

echo 4 > /proc/sys/kernel/printk
/bin/mount -t squashfs /dev/mtdblock5 /usr
/bin/mount -t squashfs /dev/mtdblock6 /mvs
/bin/mount -t squashfs /dev/mtdblock7 /ext
/bin/mount -t jffs2 /dev/mtdblock8 /mnt/mtd
dd if=/dev/zero of=/tmp/zero bs=512 count=200
losetup /dev/loop0 /tmp/zero
mkfs.vfat /dev/loop0
mkdir /tmp/ramdisk
mount /dev/loop0 /tmp/ramdisk

Если все сработало то мы загрузились в систему
Проверить работает ли СД потому что для дольнейшего нам необходимо будет делать обновление проще будет с СД
У меня СД не работало приходилось все сложнее делать.

СД можно посмотреть командой

ls /dev

и проверить есть ли /dev/mmcblk0p1

**Reply by 0003426085** [reply]
Я извиняюсь, что то Я немного запуталась, это мне надо это все вводить по очереди в командной строке или что, Я это ввожу и ничего не получается, Я извиняюсь я просто далёкий от этого 
[ВОТ ЧТО Я ВВОДИЛ.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14513140/default.txt)

**Comment by filder35** [comment]
первое вы сделали правильно загрузились в систему
второе когда система загружена и  вы вводите ( passwd root )вам пишут
Смена пароля для root
Новый пароль:
Неверный пароль: слишком много похожих символов.
Введите пароль еще раз:
Пароли не совпадают
passwd: пароль для root не изменился  (это я перевел в гугле)

Вы не сменили пароль (я ставил 12325)
вот подробнее

~ # passwd root
New password: (вводите 12345)  может не отображаться но команда идет
Retype password:  (вводите еще раз 12345)  может не отображаться но команда идет
если все правильно то должно что паспорт поменян.
либо отобразит ошибку.

  
Дальнейшие команды для монтирования файловой системы. 
они вводятся по порядку 

echo "mount all file system..."
mkdir /dev/pts
/bin/mount -av
umask 022
export runlevel prevlevel
echo "starting mdev..."
/bin/echo /sbin/mdev > /proc/sys/kernel/hotplug
mdev -s

mkdir /var/cache
mkdir /var/run
mkdir /var/log
mkdir /var/spool
echo 4 > /proc/sys/kernel/printk
/bin/mount -t squashfs /dev/mtdblock5 /usr
/bin/mount -t squashfs /dev/mtdblock6 /mvs
/bin/mount -t squashfs /dev/mtdblock7 /ext
/bin/mount -t jffs2 /dev/mtdblock8 /mnt/mtd
dd if=/dev/zero of=/tmp/zero bs=512 count=200
losetup /dev/loop0 /tmp/zero
mkfs.vfat /dev/loop0
mkdir /tmp/ramdisk
mount /dev/loop0 /tmp/ramdisk

На дальнейшие действия я составлю инструкцию поподробней.

**Reply by 0003426085** [reply]
Спасибо за буду пробовать

**Reply by 0003426085** [reply]
Я вводу пароль и выдает такое passwd: /etc/passwd: файловая система только для чтения.
passwd: невозможно обновить файл паролей /etc/passwd
[passwd root.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14513510/passwd.root.txt)

**Reply by filder35** [reply]
все тогда оставим пароль у вас он тоже не проходит
Для чего это мы все проделали  чтобы вы понимали как можно загрузить систему для редактирования  в ручном режиме.
На этом пока оставим так . Когда вы в следующий раз будете подключать камеру и пройдет лог загрузки опять надо буде производить монтирование блоков  вручную командами.

echo 4 > /proc/sys/kernel/printk
/bin/mount -t squashfs /dev/mtdblock5 /usr
/bin/mount -t squashfs /dev/mtdblock6 /mvs
/bin/mount -t squashfs /dev/mtdblock7 /ext
/bin/mount -t jffs2 /dev/mtdblock8 /mnt/mtd

**Comment by filder35** [comment]
Второй этап проверяем возможность СД видит ли его система
вставьте СД флеш в камеру
Подключаем камеру  после загрузки монтируем блоки 
echo 4 > /proc/sys/kernel/printk
/bin/mount -t squashfs /dev/mtdblock5 /usr
/bin/mount -t squashfs /dev/mtdblock6 /mvs
/bin/mount -t squashfs /dev/mtdblock7 /ext
/bin/mount -t jffs2 /dev/mtdblock8 /mnt/mtd
и смотрим 
ls /dev 
Лог выложите

**Comment by filder35** [comment]
А вот информация для изучения и понимания как работает ваша камера.
Изучая прошивку оригинал файловая структура похожа примерно на это
https://thao.pw/hacking-yoosee-camera
https://github.com/t-rekttt/yoosee-exploit/tree/main 
https://github.com/t-rekttt/yoosee-exploit/tree/main/all/all/modules/wifi/rda5995_fw  -ваш драйвер вифи
Даже может обновление которое на данном сайте может вам подойти (не уверен но можно попробовать)

Вы прошили камеру от v380
Сенсор у вас судя по фото SC2332 вифи rda5995
вот на них надо искать драйвера и пробовать запускать
У меня тоже была камера от yoosee прошивка не читалась пришлось долго возиться чтобы собрать все оборудование и чтобы оно работало.
Вот ссылки для получения информации что придется делать и пробовать.
https://github.com/EliasKotlyar/Xiaomi-Dafang-Hacks/issues/1672
  https://github.com/t-rekttt/yoosee-exploit/tree/main     --здесь кстати есть драйвер на ваш сенсор
https://ricardojlrufino.wordpress.com/2022/02/15/hack-ipcam-anyka-booting-rootfs-from-sdcard/   --запуск с СД
https://github.com/misaalanshori/misaalanshori.github.io/issues/2
https://github.com/e27-camera-hack/E27-Camera-Hack/discussions/1
https://github.com/alienatedsec/yi-hack-v5/discussions/236

**Reply by 0003426085** [reply]
3дравствуйте вот лог
[лог.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14527046/default.txt)

**Reply by 0003426085** [reply]
Я извиняюсь, что так долго не отвечал был на работе

**Reply by filder35** [reply]
СД вижу не работает это пока печально
Будем думать.
Дело в том что у вас в прошивке которая стоит на камере отсутствуют драйвера от оборудования установленного на плате камеры.  А оригинальный дамп пока не могу понять блок в котором находятся драйвера не разбирается пишет ошибку (думаю он битый поэтому у вас камера при прошивке оригинала не запускалась.)
А вы пока почитайте ссылки выше там много по камерам такого типа
Драйвера под вашу камеру и оборудование найти можно пока проблема как их на камере реализовать.
Если что вы готовы выпаять флеш ? Ну это крайний случай если не получиться так их запихать.

**Reply by 0003426085** [reply]
Хорошо Я почитаю ссылки, если что можно выпаять флеш

**Comment by filder35** [comment]
Есть вопросы.
Когда вы решили вернуть оригинальный дамп (я так понял выпаивали флешь и залили програматором) 
после включения какие симптомы были и лог вы снимали с оригинальной прошивкой?
Почему спрашиваю на той прошивке (будем называть ее v380) много может возникнуть нюансов при добавлении вашего оборудования если только найти анологичную с анологичным оборудование от v380 (это сенсор и вифи).  У меня много разных дампов от разных устройств камер анука. Но пока не нашел чтоб драйвер сенсора и вифи подходил. 
И хотелось бы узнать на чем зацикливалась прошивка оригинал при повторной заливке (я так понял подсветка включалась значит загрузчик запускал ядро и вклюяал подсветку а на чем останавливался)
И какое оригинальное приложение использовала камера

**Comment by 0003426085** [comment]
Ну Я могу, прошить обратно родной Дамп и тогда выложить Лог сюда. А приложение использовала камера 
![Screenshot_2024-03-08-07-24-51-161_com android vending-edit](https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/cc251206-21d6-4a79-9d07-f23dd8adb1c2)
Только Я это буду делать вечером, после работы.

**Reply by filder35** [reply]
Давай так и сделаем. Почему, та прошивка которая сейчас стоит она вообще другая, попробуем оригинал посмотрим на чем он останавливается и чего хочет . Если не пойдет тогда будем пробовать собирать из кусочков.

**Reply by 0003426085** [reply]
Ок, до вечера

**Reply by filder35** [reply]
Ок.
Вот здесь есть драйвера под сенсор и вифи ваш и структура вашей прошивки оригинала
https://github.com/t-rekttt/yoosee-exploit
А вот здесь занимались такой же камерой
https://thao.pw/hacking-yoosee-camera

**Reply by 0003426085** [reply]
ВОТ ЛОГ С РОДНОГО ДАМПА 
[ЛОГ ДАМПА.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14540315/default.txt)
Я ВЫПАИВАЛ И ПРОШИЛ РОДНОЙ ДАМП

**Comment by filder35** [comment]
Вот это уже лучше. Камера рабочая ни чего муденого делать не надо . Она потеряла блок где лежит все драйвера и файлы конфигурации. Попробуйте в ручную смонтировать командой

 /bin/mount -t squashfs /dev/mtdblock6 /ipc

если выдаст ошибку значит блок битый

**Reply by filder35** [reply]
логин root

попробуйте без пароля  или 123 или toor

**Reply by filder35** [reply]
Войдите в систему как root, пароль не требуется

**Reply by 0003426085** [reply]
вот что получилось
[вот что получ.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14540784/default.txt)

**Reply by 0003426085** [reply]
Наверное блок битый

**Reply by filder35** [reply]
ну все рабочее . Блок да не монтируется . Сейчас попробую на нее найти обновление. 
А вы пока проверьте СД монтируется можно посмотреть командой
ls /dev
должно быть что то вроде /dev/mmcblk0p1
если есть то все отлично  СД чистую не забудьте вставить

**Reply by 0003426085** [reply]
вродебы видит сд
[лог2.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14541191/2.txt)

**Comment by filder35** [comment]
Отформатируйте СД в fat
Разархивируйте архив с прошивкой 31.00.00.01-b.bin.enc на СД
Вставьте СД 
Включите камеру должна пойти прошивка . Только не вздумайте отключать питание во время прошивки.
Если не сработает попробуйте нажать кнопку на камере  reset
Лог потом выложите если что пойдет не так.
[Uploading 31.00.00.01-b.bin.enc.zip…]()

**Reply by 0003426085** [reply]
Я извиняюсь,не могу скачать архив, подскажите как его скачать

**Reply by filder35** [reply]
https://drive.google.com/file/d/1DbRaHeZlSDT_SlkXfNQSsR7dZ3EFOPtG/view?usp=sharing

**Reply by 0003426085** [reply]
Вроде бы все сделал так как Вы сказали, но прошивка не началась

**Reply by 0003426085** [reply]
[лог3.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14541512/3.txt)

**Reply by filder35** [reply]
покажите команду
mount

**Reply by 0003426085** [reply]
вот что получилось
[лог4.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14541579/4.txt)

**Reply by filder35** [reply]
покажите 
ls /mnt

**Reply by 0003426085** [reply]
вот лог
[лог5.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14541706/5.txt)
что делать дальше

**Reply by filder35** [reply]
https://drive.google.com/file/d/1SwWwG_Gtlfbg31zk4vBSrA76C57ODBTc/view?usp=sharing
этот файл тоже положи на СД
потом выведи команду 
ls /mnt/disc1

**Reply by filder35** [reply]
ls /mnt/disc1
должны отобразиться файлы 
31.00.00.01-b.bin.enc           ipc_gwellupdater.sh

если показало файлы выпоняем

/mnt/disc1/ipc_gwellupdater.sh

должно пойти

**Reply by 0003426085** [reply]
Надо чтобы было два файла на СД. Первый файл тоже оставить и добавить этот да

**Reply by filder35** [reply]
да два

**Reply by 0003426085** [reply]
вот лог
[лог6.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14541871/6.txt)

**Comment by filder35** [comment]
выполните
mount  /dev/mmcblk0 /mnt/disc1

проверим смонтировалось 
ls /mnt/disc1
должны отобразиться файлы 
31.00.00.01-b.bin.enc  ipc_gwellupdater.sh

если показало файлы выпоняем

/mnt/disc1/ipc_gwellupdater.sh

должно пойти

**Reply by filder35** [reply]
либо так попробуйте
mount /dev/mmcblk0p1 /mnt/disc1
потом
ls /mnt/disc1
должны отобразиться файлы
31.00.00.01-b.bin.enc ipc_gwellupdater.sh

**Reply by 0003426085** [reply]
файлы появились прошло и вот лог
[лог7.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14542055/7.txt)

**Comment by filder35** [comment]
https://drive.google.com/file/d/1obFcja7ClP1uXYeqxpeUyJvcT8GsC1i8/view?usp=sharing
пробуй поменять 31.00.00.01-b.bin.enc на который в архиве и опять прогнать команды
которыми получилось смонтировать
mount /dev/mmcblk0p1 /mnt/disc1
потом
ls /mnt/disc1
должны отобразиться файлы
upg.bin.enc      ipc_gwellupdater.sh

/mnt/disc1/ipc_gwellupdater.sh

хотя может с этим файлом сама обновиться

**Reply by 0003426085** [reply]
ВОТ ЛОГ
[лог8.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14542238/8.txt)

**Reply by filder35** [reply]
Ладно на сегодня все камера рабочая файлы обновления не видит завтра по другому попробуем

**Reply by filder35** [reply]
может флешка такая или что то мешает по идее камера сама должна смонтировать

**Reply by 0003426085** [reply]
Спасибо ВАМ большое, хорошо до завтра

**Comment by filder35** [comment]
Мы вчера маленько торопились. Надо было внимательней просматривать скрипты для обновления . Я примерно составлю новую инструкцию и выложу с файлами. И еще по СД вы ее форматируете а винда создает свои файлы в ней. Но я думаю это не сильно влияет. Хотя для линукса это может и влиять. Здесь я я не уверен. Вариантов для обновления достаточно. Камера рабочая есть  вход по СД и   UART это уже хорошо можно даже и телнет фтп  включить но это пока не нужно без вифи.

**Comment by filder35** [comment]
https://gist.github.com/faust93/8f857b2bc3f94b79ba1a275f8ed630a9
вот здесь аналог вашей камеры и даже дамп есть

**Reply by 0003426085** [reply]
Здравствуйте, Я извиняюсь что так долго не отвечал, только приехал сейчас буду пробовать

**Reply by filder35** [reply]
один пост я удалил он был не правильным. Для начала выложите 

cat /proc/mtd

**Reply by 0003426085** [reply]
вот лог
[лог9.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14546023/9.txt)

**Reply by 0003426085** [reply]
А классический вариант пробовать или нет

**Reply by filder35** [reply]
пока нет я его подправлю

**Reply by 0003426085** [reply]
Ок

**Comment by filder35** [comment]
Вот теперь можно я сначала сомневался но не знаю пройдет ли он у тебя камера и расположение блоков другое 
Вариант_1 классический

1. распаковываем содержимое архива на СД и вставляем в камеру
[updater_app.zip](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14546029/updater_app.zip)


3. запускаемся ждем окончания загрузки

4. монтируем СД

 mount /dev/mmcblk0p1 /mnt/disc1

5. проверяем монтирование и наличие нужных файлов

 ls /mnt/disc1

если файлы на месте

6. вводим команды поочереди

 chmod +x /mnt/disc1/updater
 /mnt/disc1/updater local MTD6=/mnt/disc1/app.bin

7. перезагружаемся

 reboot

выводим лог

**Reply by 0003426085** [reply]
оно постоянно одно и тое груится как его остановить, наверное циклическая переагрука
[лог10.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14546053/10.txt)

**Reply by 0003426085** [reply]
Как его оставить

**Comment by filder35** [comment]


**Reply by filder35** [reply]
по логу вроде заработала ждет подключения сенсор и вифи вроде работают

**Reply by filder35** [reply]
камеру выключи потом включи и когда лог дойдет до цикличности выключи 
а то что от начала загрузки до остановки скопируй 
посмотреть как драйвера загрузила

**Reply by filder35** [reply]
вот еще в логе я так понял к ней надо подключаться она свою сеть дает 
ssid=Petro, password=1357924680

**Reply by 0003426085** [reply]
Такое впечатление что она постоянно перезагружается. Вайфай не отображается как к ней подключиться.Постоянно бегут строки, это Так и должно быть 

https://github.com/e27-camera-hack/E27-Camera-Hack/assets/155267481/ac6dfc68-33ee-483a-a1e6-4ec2a31da3e5

Оно так постоянно как на видео

**Reply by 0003426085** [reply]
[лог 11.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14546100/11.txt)

**Reply by 0003426085** [reply]
КАК ОСТАНВИТЬ БЕГУЩИЕ СТРОКИ

**Reply by filder35** [reply]
это не перезагрузка 
закрой терминал открой поновой 
камеру выключи потом включи и когда лог дойдет до цикличности выключи
мне надо лог от U-Boot 2013.10.0-AK_V3.0.08 (Jan 07 2020 - 21:24:50)  это начала загрузки
и до цикличной информации
а то что от начала загрузки до остановки скопируй
посмотреть как драйвера загрузила

**Reply by filder35** [reply]
и на телефоне проверь вифи появилась ли новая сеть типа Free-AP1 или Free-AP2 или Petro если да то подключись к ней пароль выше

**Reply by filder35** [reply]
либо пробуй приложением подключиться

**Comment by filder35** [comment]
либо пробуй сделать reset сброс до зоводских кнопка рядом с СД

**Comment by 0003426085** [comment]
Сделал сброс сеть появилась,Я подключился но видео нет пишет что сеть занятая

**Reply by 0003426085** [reply]
УРА-А-А  ПОЯВИЛАСЬ КАРТИНКА

**Comment by 0003426085** [comment]
НО ПОЧЕМУТО ГОРИТ  БЕЛАЯ ПОДСВЕТКА. СПАСИБО ВАМ ОГРОМНОЕ ВЫ НАСТАЯЩИЙ МАСТЕР СВОЕГО ДЕЛА

**Reply by 0003426085** [reply]
Спасибо, спасибо, спасибо ВАМ

**Reply by filder35** [reply]
Отлучался маленько. Вот и победили. По подсветке подсказать не могу. Может где в приложении это режим день ночь. А лучше скиньте полный лог от загрузчика до подключения надо смотреть что не срабатывает.

**Reply by 0003426085** [reply]
вот лог
[лог12.txt](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14546330/12.txt)

**Reply by 0003426085** [reply]
Все с подсветкой разобрался. Я уже её полностью собрал. Вроде бы работает хорошо. Спасибо ВАМ ещё раз, Вы мастер своего дела спасибо

**Reply by filder35** [reply]
Все пользуйтесь. Учите Линукс винда для слабаков.

**Reply by 0003426085** [reply]
Ок, спасибо. Счастья, здоровья, любви и успехов Вам

**Reply by 0003426085** [reply]
Я извиняюсь, а какие команды используют ещё, если не сложно можете выложить и объяснить, может мне пригодится на будущее в использовании и изучении Линукса, Я ВАМ очень благодарен, спасибо ВАМ большое Вы один за год смогли Мне помочь,Вы конечно красава спасибо

**Comment by filder35** [comment]
линукс это не команды это система
[Embedded_Linux_system_design_and_development_ru.pdf](https://github.com/e27-camera-hack/E27-Camera-Hack/files/14548378/Embedded_Linux_system_design_and_development_ru.pdf)
https://linuxmint-installation-guide.readthedocs.io/ru/latest/
https://serverspace.ru/about/blog/bazovye-komandy-linux-v-terminale/?utm_source=google.com&utm_medium=organic&utm_campaign=google.com&utm_referrer=google.com

**Reply by 0003426085** [reply]
Спасибо, буду изучать
